macaw-aarch32/tests/arm/test-empty-section.c

```diff
@@ -0,0 +1,14 @@
+// test.c
```

Comment:
Can we add a reference to #302 here in this file so that someone browsing this code can get more details on the expected effects of building this test?